pipeline/tasks/packman_tasks.py

```diff
@@ -23,7 +23,6 @@
 class PackmanTaskBase(task_base.TaskBase):
     def run_packman(self, language, api_name, *additional_args):
         # Fix the api_name convention (ex. logging-v2) for packman.
-        api_name = api_name.replace('-', '/')
```

Comment:
I like to have the same logic into this base class if all of the subclasses are doing same thing.
Why not invoking packman_api_name() here?